src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setFigureType("circle");
+        setColor(color);
+        calculateArea();
+    }
+
+    @Override
+    public void calculateArea() {
+        setArea(Math.round(Math.PI * Math.pow(radius, 2)));
+    }
+
+    @Override
+    public String toString() {
+        return super.toString()
+                + getFigureType()
+                + ", area: "
+                + getArea()
+                + " sq.units"
+                + ", radius: "
+                + radius
+                + " units, color: "
+                + getColor();
```

Comment:
From the requirements:

```
To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this)
```

So, create an interface with one abstract method `draw()` and implement it in your `Figure`.